utils/elect-rc-leader

```diff
@@ -47,6 +47,14 @@ get_leader_node() {
         jq -r '.[] | .Node'
 }
 
+get_node_name() {
```

Comment:
Same function name with same purpose in two files.
Can we have only one at one place?